<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BandRatioDialog</class>
 <widget class="QDialog" name="BandRatioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Spectral Region of Interest</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Wavelength Range</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Region 1 Left Bound</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="firstMaxLineEdit">
       <property name="placeholderText">
        <string>Region 1 Left Bound</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Region 1 Right Bound</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="firstMinLineEdit">
       <property name="placeholderText">
        <string>Region 1 Right Bound</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="secondMinLineEdit">
       <property name="placeholderText">
        <string>Region 2 Left Bound</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="secondMaxLineEdit">
       <property name="placeholderText">
        <string>Region 2 Right Bound</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="rangeLabel">
       <property name="text">
        <string>200-2000 cm-1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Region 2 Left Bound</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Region 2 Right Bound</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCustomPlot" name="spectrumPlot" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Map Settings</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>67</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Color Gradient</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Peak Determination</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="peakComboBox">
       <item>
        <property name="text">
         <string>Intensity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Area</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Derivative</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bandwidth (FWHM)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="gradientComboBox">
       <property name="toolTip">
        <string>An asterix by the color scheme name indicates a divergent color scheme particularly suited for this kind of map.</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>ColorBrewer BlueGreen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer BluePurple</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer GreenBlue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer OrangeRed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer PurpleBlue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer PurpleBlueGreen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer PurpleRed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer RedPurple</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer YellowGreen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer YellowGreenBlue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer YellowOrangeBrown</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer YellowOrangeRed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer Blues</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer Greens</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer Oranges</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer Purples</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer Reds</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ColorBrewer Grayscale</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QCustomPlot Grayscale</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QCustomPlot Night</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QCustomPlot Candy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QCustomPlot Ion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QCustomPlot Thermal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>↔ QCustomPlot Polar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>↔ QCustomPlot Spectrum</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QCustomPlot Jet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QCustomPlot Hues</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QCustomPlot Hot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QCustomPlot Cold</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>↔ ColorBrewer BrownBlueGreen </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>↔ ColorBrewer PinkYellowGreen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>↔ ColorBrewer PurpleGreen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>↔ ColorBrewer PurpleOrange</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>↔ ColorBrewer RedBlue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>↔ ColorBrewer RedGray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>↔ ColorBrewer RedYellowBlue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>↔ ColorBrewer RedYellowGreen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>↔ ColorBrewer Spectral</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>↔ Vespucci Spectral</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Map Name</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>20</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BandRatioDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BandRatioDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
